code/modules/bitrunning/server/obj_generation.dm

```diff
@@ -129,54 +129,34 @@
 
 /// Scans over neo's contents for bitrunning tech disks. Loads the items or abilities onto the avatar.
 /obj/machinery/quantum_server/proc/stock_gear(mob/living/carbon/human/avatar, mob/living/carbon/human/neo, datum/lazy_template/virtual_domain/generated_domain)
-	var/domain_forbids_items = generated_domain.forbids_disk_items
-	var/domain_forbids_spells = generated_domain.forbids_disk_spells
+	var/domain_forbids_flags = generated_domain.external_load_flags
 
 	var/import_ban = list()
 	var/disk_ban = list()
-	if(domain_forbids_items)
+	if(domain_forbids_flags & DOMAIN_FORBIDS_ITEMS)
 		import_ban += "smuggled digital equipment"
 		disk_ban += "items"
-	if(domain_forbids_spells)
+	if(domain_forbids_flags & DOMAIN_FORBIDS_ABILITIES)
 		import_ban += "imported_abilities"
 		disk_ban += "powers"
 
 	if(length(import_ban))
-		to_chat(neo, span_warning("This domain forbids the use of [english_list(import_ban)], your disk [english_list(disk_ban)] will not be granted!"))
+		to_chat(neo, span_warning("This domain forbids the use of [english_list(import_ban)], your externally loaded [english_list(disk_ban)] will not be granted!"))
 
-	var/failed = FALSE
+	var/return_flags = NONE
 
-	// We don't need to bother going over the disks if neither of the types can be used.
-	if(domain_forbids_spells && domain_forbids_items)
-		return
-	for(var/obj/item/bitrunning_disk/disk in neo.get_contents())
-		if(istype(disk, /obj/item/bitrunning_disk/ability) && !domain_forbids_spells)
-			var/obj/item/bitrunning_disk/ability/ability_disk = disk
+	// We don't need to bother going over the items if nothing can used.
+	//if(domain_forbids_flags)
+	//	return
+	//for(var/obj/item/potential_sideloader in neo.get_contents())
+	//	return_flags |= SEND_SIGNAL(potential_sideloader, COMSIG_BITRUNNER_STOCKING_GEAR, avatar, neo, domain_forbids_flags)
```

Comment:
why is this still here?

Reply:
I could _swear_ I removed this on a passover
thanks